www/audit.py

```diff
@@ -1,6 +1,6 @@
 from www import app
 from .db import database, User, Feature, Project, Task, fn_Random
-from .util import update_features, update_audit, update_features_cache
+from .util import update_features, update_audit, update_features_cache, update_csv, read_csv
```

Comment:
`util`-Module sind zu vermeiden, da sie schnell zu einem Sammelbecken für alles mögliche werden, ohne dass einem der Namen Informationen darüber liefert, für was alles. Siehe YouTube-Video [Naming Things in Code](https://youtu.be/-J3wNP6u5YU).

Dieses hier bestand bereits, aber lass es uns nicht zusätzlich füllen. Ob die neuen Funktionen stattdessen in dieses Modul hier (`www.audit`) gehören oder in ein neues, sinnvoll zu benennendes (dass dieses relativ klein wäre, macht nichts), kann ich auf den ersten Blick nicht beurteilen.